### PR TITLE
Calculator components

### PR DESCRIPTION
**Pull request by JAAR91**
- Delete all the boilerplate from CRA (text, images, styles).
- Create inside components the new Calculator.js.
- Use class-based components.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    ans = <button className="col-3 p-5 bg-white" type="button">{text}</button>;
+  }
+  if (text === '0') {
+    ans = <button className="col-6 p-5 bg-white" type="button">{text}</button>;
+  }
+
+  return ans;
+};
+
+export default CalcButton;
```
Comment (arslanbisharat) [inline src/components/Calculator.js]: - [x] Let's make calculator.js a class-based component. You have done this in App.js let's make it here too.

File: src/home.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  ReactDOM.render(
+    <div className="p-5 mt-5">
+      <h1 className="mt-5">Welcome to our page!</h1>
+      <p className="mt-3 fs-4">
+        Loerem ipsum dolor sitamet, consectetur adipisicing elit, sed do eiusmod tempor incididunt
+        ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+        ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in
+        reprehenderit it voluptate velot esse cillum dolore eu fugiat nilla pariatur. Excepteur
+        sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id
+        est laborum. Ipsu.
```
Comment (arslanbisharat) [inline src/home.js:14]: - [x] Please add a descriptive Readme in your project. Currently your readme is not descriptive .

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/App.js [hunk 3]
```diff
@@ -1,29 +1,40 @@
...
+        <CalcButton text="0" Bclass="true" />
+        <CalcButton text="." Bclass="true" />
+        <CalcButton text="=" Bclass="false" />
+      </div>
+    );
+    return value;
+  }
+}
 
 function App() {
```
Comment (Dipeshtwis) [inline src/App.js:35]: [OPTIONAL]

- In this milestone, you are practicing the class component, So I would recommend changing `App` from `function` to `class` component.

**Review by Dipeshtwis — APPROVED** [review]
# Hi @JAAR91, 

### Status: Approved ✔️

:+1:  You have implemented all the requested changes suggested by the previous reviewer.

Your project is complete!  There are some optional changes I've suggested to you to make your project best, other than that nothing else to say.
Congratulations! 🎉 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**_Check the comments under the review_**


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Dipesh Kumar_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**